### PR TITLE
Extending SMGL Serializer to support more banks

### PR DESCRIPTION
**Pull request by ghost**
- Bumping up version from 102 to 103
- Toleration \r\n in received documents

### CODE REVIEW

File: src/Mocoding.Ofx/Serializers/OfxSgmlSerializer.cs [hunk 1]
```diff
@@ -48,6 +48,8 @@ public override string Serialize(OFX model)
         public override OFX Deserialize(string inputString)
         {
             // convert sgml to xml using Regex
+            inputString = inputString.Replace("\r\n", "");
```
Comment (offbeatful) [inline src/Mocoding.Ofx/Serializers/OfxSgmlSerializer.cs:51]: Can you please provide better explanation around this?

**Comment by offbeatful** [comment]
Everything looks great! Thanks for contribution.
Please fix the test and I am not clear on \r\n thing. The tests may fail under windows but I am working on providing development in container feature.

**Comment by ghost** [comment]
> Please fix the test and I am not clear on \r\n thing. 

I tested against Citibank and it was failing because their response contained a bunch of "\r\n". The regex would not handle it. Perhaps I could amend the regular expression, but I did not want to make it unnecessary complex.

> The tests may fail under windows but I am working on providing development in container feature.

I fixed the version and I assume it's now broken for the reason you mentioned.

Also I should mention why I made the new public method: the accounting software I use (proprietary) submits "102" for the version, and banks reject it. So I'm using your OFX library in my proxy that takes requests from the software towards my proxy, then replaces the Auth part of the payload with a working one, sends it with the rest of the original payload to the bank and returns results back to the software. Works like a charm.

**Comment by offbeatful** [comment]
Sounds very good! I assume this will also allow users to send custom requests (like pay credit cards, query for due dates etc). OFX protocol has much more capabilities...

'\r\n' use case is well understood. Thanks again for contribution.

**Comment by offbeatful** [comment]
Just reviewed the pipeline - it says "The total line coverage is below the specified 80". I would appreciate if you add a test for PrepareOfxRequest

**Review by offbeatful — APPROVED** [review]

**Pull request merged**